Issue#129 fix styles for selector

ihordubas1 opened the pull request:
#129

Commit:
Add a change of mixins

kfarr committed:
Merge branch 'master' into issue#129

Commit:
Merge master

Commit:
Fix styles for selector

kfarr committed:
Merge branch 'issue#129' of https://github.com/3DStreet/3dstreet-editor into issue#129

kfarr commented on src/components/components/Mixins.js:
```diff
@@ -42,7 +42,7 @@ export default class Mixin extends React.Component {
   updateMixins = (value) => {
     const entity = this.props.entity;
-
+    value = Array.isArray(value) ? value : [value];
```
what is this for? @ihordubas1

kfarr replied:
reverted

kfarr replied:
this looks like a way to handle a case where a non-array is passed; that shouldn't happen with the new version, reverting

ihordubas1 replied:
@kfarr 
can you check the new PR (https://github.com/3DStreet/3dstreet-editor/pull/183) 
I think that we can close the current PR and use #183 .

kfarr committed:
revert to remove unused code

@ihordubas1 you added this back in but they were not used in the new version, I am deleting them again

kfarr committed:
revert icons

kfarr committed:
revert mixins

remove legacy import `Selector`

kfarr committed:
revert mixins

kfarr commented:
use https://github.com/3DStreet/3dstreet-editor/pull/183 instead
